templates/InstallChocolatey.ps1.erb

```diff
@@ -99,8 +101,7 @@ $chocInstallPS1 = Join-Path $toolsFolder "chocolateyInstall.ps1"
 
 if ($PSVersionTable.psversion.Major -gt 2) {
   & $chocInstallPS1
-}
-else {
+} else {

```

Comment:
You wouldn't think that line breaks would cause such a big deal. Redirecting input one line at a time FTL. 
Ha!